terraform/globals.tf

```diff
@@ -149,3 +149,8 @@ variable "assets_prefix" {
   description = "Prefix for global assests like S3 buckets"
   default     = "gds-paas"
 }
+
+variable "deploy_roadmap" {
```

Comment:
I suppose we don't and do in the same time. We don't really need it to be deployed to staging or master. On some occasion, you'd like to deploy it to dev just to see if it works or checkout the future changes.